examples/gaussian.py

```diff
@@ -15,6 +15,7 @@
 model = A*exp(-((x - x0)**2/(2 * sig**2)))
 
 # Sample 10000 points from a N(15.0, 1.5) distrubution
+np.random.seed(seed=123456789)
```

Comment:
@Jhsmit Do you know if providing a better guess would already solve the problem? If that alone is not enough I will merge this PR with the seed but I would prefer a fix without the seed.